src/libs/modulegroups.c

```diff
@@ -2628,6 +2660,36 @@ static gboolean _manage_direct_basic_popup(GtkWidget *widget, GdkEventButton *ev
   return FALSE;
 }
 
+static void _manage_direct_full_active_toggled(GtkWidget *widget, dt_lib_module_t *self)
+{
+  dt_lib_modulegroups_t *d = (dt_lib_modulegroups_t *)self->data;
+  d->full_active = gtk_check_menu_item_get_active(GTK_CHECK_MENU_ITEM(widget));
+  const int cur = d->current;
+  _manage_direct_save(self);
+  d->current = cur;
+  _lib_modulegroups_update_iop_visibility(self);
+}
+
+static gboolean _manage_direct_active_popup(GtkWidget *widget, GdkEventButton *event, dt_lib_module_t *self)
+{
+  if(event->type == GDK_BUTTON_PRESS && event->button == 3)
+  {
+    dt_lib_modulegroups_t *d = (dt_lib_modulegroups_t *)self->data;
+    GtkWidget *pop = gtk_menu_new();
+    gtk_widget_set_name(pop, "modulegroups-popup");
+
+    GtkWidget *smt = gtk_check_menu_item_new_with_label(_("show all module in history"));
```

Comment:
```suggestion
    GtkWidget *smt = gtk_check_menu_item_new_with_label(_("show history modules in active group"));
```

I'd like something shorter but I think that might be the shortest I can come up with at the moment

Reply:
Same in the other instance of this checkbox too

Reply:
hmm... are you sure the last part is really needed ? the item is only shown after a right click on the active group icon, so imho we can presume that the user know that the option will be apply there... (but I'm not against your suggestion either)

Reply:
Ah ok then that makes sense. So this checkbox should say "show all history modules" (and should have a tooltip the same as the other) and the presets window should have the label "show all history **modules** in active group"